refactor: restructure footer layout

Aksshay88 opened the pull request:
This pull request includes significant updates to various components of the application, focusing on enhancing the footer with detailed one and to easily navigate when clicked in the footer .

![image](https://github.com/user-attachments/assets/9fef426e-1a26-457c-a5c6-82f3b1d2265c)

Resolves issues #636

Aksshay88 commented:
@AceTheCreator Pls review this one .

AceTheCreator reviewed (changes requested):
Please kindly focus on the existing footer section in this PR, the additional changes introduced in this footer needs further discussions...

thulieblack commented:
Please remove all the copyright information

Aksshay88 commented:
Yeah sure will remove it 👍

Aksshay88 commented:
Removed the copyright stuff . 
You could review it pls

thulieblack commented:
@Aksshay88 I don't think we need this 
![image](https://github.com/user-attachments/assets/808ae3b7-f68f-4d4d-8aac-53e56d4b48ea)

Aksshay88 commented:
@thulieblack okay then I remove that part 
So we gonna have the[ Follow us and Made with :heart: by asyncapi community alone ]right ? 
will change it and give a PR for this one

ashmit-coder commented:
> @thulieblack okay then I remove that part So we gonna have the[ Follow us and Made with ❤️ by asyncapi community alone ]right ? will change it and give a PR for this one

Hey @Aksshay88 is this request being worked upon?

Aksshay88 commented:
> > @thulieblack okay then I remove that part So we gonna have the[ Follow us and Made with ❤️ by asyncapi community alone ]right ? will change it and give a PR for this one
> 
> Hey @Aksshay88 is this request being worked upon?

@ashmit-coder yeah I am working on this one will give a Pr by today 
Thank you !!!

Aksshay88 commented:
@thulieblack and @ashmit-coder I kindly request you to review the pr .

I have removed the Subscription stuff from the landing page where it doesn't looks good over there . So thought of incorporating it in the footer where If I just kept the Follow us alone in the footer , it looks very simple and that is why I have added the subscription component in the footer along with the follow us in the same Footer Component 

I have made the changes according the suggestion you guys have given . 

Any changes I am willing to do .  
Thank you !!! 


I have attached the Screenshot for the above PR . Pls do look at it : 

![image](https://github.com/user-attachments/assets/3f85a010-e150-401c-9e97-4aee4904b9af)

thulieblack commented:
Rather do this, coz this isn't looking nice

![image](https://github.com/user-attachments/assets/ef355914-5f7c-4884-adcf-073968d9437f)

Also where is the Code of Conduct?

Aksshay88 commented:
Sure then will move the Follow us in the  vertical format and include the code of conduct below that one 👍

Aksshay88 commented:
@thulieblack 
![image](https://github.com/user-attachments/assets/31c69388-2a1c-498a-87d8-585fde173e2f)


Is this okay ? 
Or Any changes , I am willing to do 
Thank you !!!
